app/services/axios.ts

```diff
@@ -57,6 +58,8 @@ instance.interceptors.response.use(
 
 export const handleAxiosError = (err: AxiosError): [null, AxiosError] => {
   let errorMessage = null;
+  console.log(err);
```

Comment:
do we want this console log to stay here

Reply:
Good catch, removed it